content/shared/influxdb3-cli/create/database.md

```diff
@@ -69,4 +70,14 @@ Flags override their associated environment variables.
 influxdb3 create database --token AUTH_TOKEN DATABASE_NAME
 ```
 
+### Create a database with a retention period
```

Comment:
Just FYI, not a suggestion: I've used 3+ examples as my threshold for adding an examples TOC. We could add one here, but it's not a big deal.